homework/homework.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+	"gocourse05/core/camera"
+	"math/rand/v2"
+)
+
+type ID interface {
+	GetId() int
```

Comment:
```suggestion
	ID() int
```

https://go.dev/doc/effective_go#Getters
https://go.dev/wiki/CodeReviewComments#initialisms

Reply:
звикаю по трохи до такого неймінгу